packages/adapter/src/lib/adapter/adapter.ts

```diff
@@ -2367,9 +2367,26 @@ export class AdapterClass extends EventEmitter {
         return this._updateConfig({ newConfig });
     }
 
-    private async _updateConfig(options: InternalUpdateConfigOptions): ioBroker.SetObjectPromise {
+    private async _updateConfig(options: InternalUpdateConfigOptions, useAutoEncryption?: boolean): ioBroker.SetObjectPromise {
         // merge the old and new configuration
-        const _config = Object.assign({}, this.config, options.newConfig);
+        let _config = Object.assign({}, this.config, options.newConfig);
+
+        if(useAutoEncryption) {
```

Comment:
the boolean is unneeded because I would always check the new data for the fields and encrypt if needed

Reply:
I added it for backwards compatibility. If that's not necessary, I'll remove it again :-)